sdk/datafactory/Azure.ResourceManager.DataFactory/src/Customized/AvroDataset.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text.Json;
+using Azure.Core;
+using Azure.Core.Expressions.DataFactory;
+
+namespace Azure.ResourceManager.DataFactory.Models
```

Comment:
Better to split this file into `AvroDataset.cs` and `AvroDataset.Serliaization.cs`.